src/transformers/models/clip/processing_clip.py

```diff
@@ -0,0 +1,170 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Image/Text processor class for CLIP
+"""
+from ...tokenization_utils_base import BatchEncoding
+from .feature_extraction_clip import CLIPFeatureExtractor
+from .tokenization_clip import CLIPTokenizer
+
+
+class CLIPProcessor:
+    r"""
+    Constructs a CLIP processor which wraps a CLIP feature extractor and a CLIP tokenizer into a single processor.
+
+    :class:`~transformers.CLIPProcessor` offers all the functionalities of :class:`~transformers.CLIPFeatureExtractor`
+    and :class:`~transformers.CLIPTokenizer`. See the :meth:`~transformers.CLIPProcessor.__call__` and
+    :meth:`~transformers.CLIPProcessor.decode` for more information.
+
+    Args:
+        feature_extractor (:obj:`CLIPFeatureExtractor`):
+            An instance of :class:`~transformers.CLIPFeatureExtractor`. The feature extractor is a required input.
```

Comment:
```suggestion
            The feature extractor is a required input.
```
You said it's an instance of that on the previous line ;-)